time/AddTaskViewController.swift

```diff
@@ -66,27 +66,90 @@ class AddTaskViewController: UIViewController, UITextFieldDelegate {
         let blue = randomColors[rng].blue!
         
         print("\(red), \(green), \(blue)")
+        let projectExists = doesProjectExist(name: name)
+        
+        if (projectExists) {
+            // alert user that the name aleready exists
+            print("YOU HAVE THE SAME NAME AS AN EXISTING PROJECT NAME")
```

Comment:
Not sure there is a need for the printed message to be different to the one seen by the user ("that project name already exists") but great functionality :tada: